apps/system/js/download/download_manager.js

```diff
@@ -47,16 +49,59 @@ var DownloadManager = (function() {
         window.addEventListener('will-shutdown', function onShutdown() {
           ev.download.pause();
         });
+      })['catch'](function(err) {
```

Comment:
This is a `gjslint` workaround :'( . Maybe we should add a `// XXX workaround gjslint` here?